tests/Unit/Evolution/Systems/ScalarWave/Test_Equations.cpp

```diff
@@ -249,5 +250,95 @@ SPECTRE_TEST_CASE("Unit.Evolution.Systems.ScalarWave.UpwindFlux",
   check_upwind_flux<3>(3, time);
 }
 
+namespace {
+template <size_t Dim>
+void check_penalty_flux(const size_t num_pts) {
+  const Scalar<DataVector> used_for_size{pow<Dim>(num_pts)};
```

Comment:
To avoid repeating the `pow` as many times, I suggest:
- call the input argument `num_pts` -> `pts_per_dim`
- add a new line above this one `const size_t num_pts = pow<Dim>(pts_per_dim);`
- replace here and below `pow<Dim>(num_pts)` -> `num_pts`

Reply:
changed